### PR TITLE
refactor(backend): use LogError on contributionMessageResolver

### PR DESCRIPTION
**Pull request by ulfgebhardt**
## 🍰 Pullrequest

fix logger on contributionMessageResolver

### Issues

- contributes to #2487 
- contributes to #1870 
- based on #2662

### Todo

- [X] None

### CODE REVIEW

File: backend/src/graphql/resolver/ContributionLinkResolver.ts [hunk 1]
```diff
@@ -39,35 +40,30 @@ export class ContributionLinkResolver {
     }: ContributionLinkArgs,
   ): Promise<ContributionLink> {
     isStartEndDateValid(validFrom, validTo)
+    // TODO: this should be enforced by the schema.
```
Comment (Mogge) [inline backend/src/graphql/resolver/ContributionLinkResolver.ts:43]: Same as in contribution link resolver. `name` can be `''` and therefore evaluates to true. But this is checked later in line 47

File: backend/src/graphql/resolver/ContributionLinkResolver.ts [hunk 2]
```diff
@@ -39,35 +40,30 @@ export class ContributionLinkResolver {
...
-      const msg = `The value of 'name' with a length of ${name.length} did not fulfill the requested bounderies min=${CONTRIBUTIONLINK_NAME_MIN_CHARS} and max=${CONTRIBUTIONLINK_NAME_MAX_CHARS}`
-      logger.error(`${msg}`)
-      throw new Error(`${msg}`)
+    if (name.length < CONTRIBUTIONLINK_NAME_MIN_CHARS) {
+      throw new LogError('The value of name is too short', name.length)
     }
+    if (name.length > CONTRIBUTIONLINK_NAME_MAX_CHARS) {
+      throw new LogError('The value of name is too long', name.length)
+    }
+    // TODO: this should be enforced by the schema.
```
Comment (Mogge) [inline backend/src/graphql/resolver/ContributionLinkResolver.ts:53]: No, see above

File: backend/src/graphql/resolver/ContributionMessageResolver.ts [hunk 3]
```diff
@@ -149,8 +144,7 @@ export class ContributionMessageResolver {
       await queryRunner.commitTransaction()
     } catch (e) {
       await queryRunner.rollbackTransaction()
-      logger.error(`ContributionMessage was not successful: ${e}`)
-      throw new Error(`ContributionMessage was not successful: ${e}`)
+      throw new LogError('ContributionMessage was not sent successfully', e)
```
Comment (Mogge) [inline backend/src/graphql/resolver/ContributionMessageResolver.ts:147]: May be at this point it is too few information? Should the original error message be passed to the frontend as well?

**Review by Mogge — DISMISSED** [review]
Please get rid of to dos. I think at this point we pass not enough information to the frontend

File: backend/src/graphql/resolver/ContributionMessageResolver.ts (hunk 3 above)
Comment (ulfgebhardt) [reply backend/src/graphql/resolver/ContributionMessageResolver.ts]: I agree, fixed

**Review by Mogge — APPROVED** [review]

**Pull request merged**